index.tt

```diff
@@ -6,152 +6,251 @@
-      </a>
-      News
-    </h2>
-    [% INSERT "latest-news.xhtml" %]
-    <p><a href="news.html" class="btn btn-mini">More »</a></p>
+  <div class="span4">
+    <h2>Reproducible</h2>
+    <p>
+      Nix builds packages in isolation from each other. This ensures that they
+      are reproducible and don't have undeclared dependencies, so <strong>if a
+        package works on one machine, it will also work on another</strong>.
+    </p>
   </div>
-  <div class="span6">
-
-    <h2>
-      <a href="/blogs.xml">
-        <img src="/logo/rss.png"/>
-      </a>
-      Blog Posts
-    </h2>
-    <ul class="nixos-blogs">
-      [% FOREACH b IN JSON.Escape.json_decode(IO.All.new('blogs.json').all).channel.item.first(5); IF b; %]
-        <li>
-          <div class="nixos-blog">
-            [%
-              USE title = String(HTML.escape(b.title.remove('^[^:]*:').trim));
-              title = title.replace('^[a-z]', title.copy.truncate(1).upper)
-              author = b.title.split(':').0.trim;
-            %]
-            [%
-            %]
-            <div class="nixos-blog-title">
-              <a [% HTML.attributes(href => b.link) %]>[% title %]</a>
-            </div>
-            <div class="nixos-blog-author-info">
-              <span class="nixos-blog-author">[% HTML.escape(author) %]</span>
-              <span class="muted"> wrote on [% HTML.escape(b.pubDate) %]</span>
-            </div>
-          </div>
-        </li>
-      [% END; END %]
-      <p><a href="https://planet.nixos.org/" class="btn btn-mini">More »</a></p>
-    </ul>
-
-    <h3>Twitter</h3>
+  <div class="span4">
+    <h2>Declarative</h2>
     <p>
-      <a class="twitter-timeline" data-chrome="noheader nofooter transparent" href="https://twitter.com/nixos_org" data-widget-id="433975692670152704">Tweets by @nixos_org</a>
-      <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
+      Nix makes it <strong>trivial to share development and build
+        environments</strong> for your projects, regardless of what programming
+      languages and tools you’re using.
     </p>
+  </div>
+  <div class="span4">
+    <h2>Reliable</h2>
+    <p>
+      Nix’s ensures that installing or upgrading one package <strong>cannot
+        break other packages</strong>. It allows you to <strong>roll back to
+        previous versions</strong>, and ensures that no package is in an
+      inconsistent state during an upgrade.
+    </p>
+  </div>
+</div>
+
+
+<hr />
 
+<div id="packages-search" class="row-fluid text-center">
+  <div class="span12">
+    <h1>Choose from Thousands of Packages</h1>
+    <p class="text-center lead">
+      The Nix Packages collection (<a href="https://github/NixOS/nixpkgs">nixpkgs</a>) is a set of
```

Comment:
This text might be a bit too technical but I took it from nixpkgs about page. If somebody comes up with something that would make it obviously better I would change it in this PR, if not I think it is fine for first round to just use what we were already using.

Reply:
Remove the license part

1. it just looks better when shorter
2. it's a detail not importent here, when we want to show we have many packages and let viewers search

better put "open source" or even better "free software" in some of the first sentences of the page

![Screenshot from 2020-05-20 19-02-02](https://user-images.githubusercontent.com/91113/82475458-cfe33000-9acc-11ea-92b3-5df1a3d789a2.png)